client/app/scripts/liveblog-marketplace/actions/marketplace.js

```diff
@@ -0,0 +1,67 @@
+liveblogMarketplace
+    .factory('MarketplaceActions', ['Dispatcher', 'api', '$http', 'lodash',
+        function(Dispatcher, api, $http, _) {
+            return {
+                getBlogs: function(filters) {
+                    api.get('/marketplace/blogs', { where: filters })
+                        .then(function(blogs) {
+                            Dispatcher.dispatch({
+                                type: 'ON_GET_BLOGS',
+                                blogs: blogs,
+                                filters: filters
+                            });
+                        });
+                },
+                getMarketers: function() {
+                    api.get('/marketplace/marketers').then(function(marketers) {
+                        Dispatcher.dispatch({
+                            type: 'ON_GET_MARKETERS',
+                            marketers: marketers
+                        });
+                    });
+                },
+                toggleFilter: function(filters, type, value) {
+                    if (filters.hasOwnProperty(type) && filters[type] == value)
```

Comment:
use `===`
and I think is better to have just `filters && filters[type] === value`.